pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/strategy/DefaultNamespaceBundleSplitStrategyImpl.java

```diff
@@ -98,9 +98,8 @@ public Set<SplitDecision> findBundlesToSplit(LoadManagerContext context, PulsarS
 
             if (!channel.isOwner(bundle)) {
                 if (debug) {
-                    log.error(String.format(CANNOT_SPLIT_BUNDLE_MSG
+                    log.warn(String.format(CANNOT_SPLIT_BUNDLE_MSG
                             + " This broker is not the owner.", bundle));
-                    counter.update(Failure, Unknown);
```

Comment:
Do we need update counter?

Reply:
No. This can happen a lot, and we don't want to add noise in failure count.